### PR TITLE
chore(wren-ai-service): add script for mdl properties preparation

### PR DESCRIPTION
**Pull request by cyyeh**
add a script under the eval folder for mdl properties preparation
- generating csv file that is easier for non-technical users to update mdl properties
- filling in mdl properties defined in csv file to mdl json file

### CODE REVIEW

File: wren-ai-service/eval/preparation.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    with open(f"{Path(mdl_path).stem}_new.json", "w") as file:
+        file.write(orjson.dumps(mdl).decode())
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--mdl_path", type=str, help="Path to the MDL JSON file", required=True
+    )
+    parser.add_argument("--csv_path", type=str, help="Path to the input CSV file")
```
Comment (paopa) [inline wren-ai-service/eval/preparation.py:103]: nitpick: I recommend avoiding the use of both hyphens and underscores in the option. Personally, I prefer using hyphens.

File: wren-ai-service/eval/preparation.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import argparse
+from pathlib import Path
+
+import orjson
+
+
+def gen_eval_preparation_data_from_json_to_csv(mdl_path: str):
+    assert Path(mdl_path).exists(), f"File not found: {mdl_path}"
```
Comment (paopa) [inline wren-ai-service/eval/preparation.py:8]: assertions can be disabled with the -O option or by setting the PYTHONOPTIMIZE environment variable to a non-empty string. If we expect this behavior to extend beyond debugging purposes, using a conditional statement might be a better approach.

See also:
- https://docs.python.org/3/using/cmdline.html#cmdoption-O
- https://docs.python.org/3/using/cmdline.html#envvar-PYTHONOPTIMIZE

File: wren-ai-service/eval/preparation.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            )
+
+    with open(f"{Path(mdl_path).stem}.csv", "w") as file:
+        for row in csv_data:
+            file.write(",".join(row) + "\n")
+
+
+def gen_new_mdl_from_csv(mdl_path: str, csv_path: str):
+    assert Path(mdl_path).exists(), f"File not found: {mdl_path}"
+    assert Path(csv_path).exists(), f"File not found: {csv_path}"
```
Comment (paopa) [inline wren-ai-service/eval/preparation.py:43]: A gentle mention, the same as the above comment

**Review by paopa — APPROVED** [review]
LGTM. have a few nitpick comments tho. not all need to be done. 😆

File: wren-ai-service/eval/preparation.py (hunk 1 above)
Comment (cyyeh) [reply wren-ai-service/eval/preparation.py]: adjusted

File: wren-ai-service/eval/preparation.py (hunk 2 above)
Comment (cyyeh) [reply wren-ai-service/eval/preparation.py:8]: I'll keep it as of now

**Pull request merged**